orchestrator/src/main/java/com/scottlogic/datahelix/generator/orchestrator/generate/GenerateCommandLine.java

```diff
@@ -65,6 +66,12 @@ private void checkForAlphaGenerationDataTypes() {
         }
     }
 
+    @CommandLine.Option(
+        names = { "-V", "--version" },
+        versionHelp = true,
+        description = "Print version information and exit.")
+    boolean versionRequested;
+
```

Comment:
Here a capital letter is used for the single letter option but all other single letter options are lower case. 

It would probably best to make them all consistant

Reply:
I'm sure I have seen `-V` used as the single letter "print the version" command elsewhere.  The programs I have just looked up - git and gnu coreutils - both only use `--version` without a single-letter equivalent.

Reply:
(`-V` is listed as the preferred option for "print the version" in _The Art Of Unix Programming_ by Eric S Raymond)